Testscripts/Linux/utils.sh

```diff
@@ -3554,6 +3558,10 @@ function get_OSdisk() {
 # Sets the $PLATFORM variable to one of the following: Azure, HyperV
 # Takes no arguments
 function GetPlatform() {
+	which route
```

Comment:
Its better to use ```command -v``` to determine the existence of utility. Its built-in in shell.